src/Umbraco.Web.UI.Client/src/views/components/tabs/umb-tabs-nav.html

```diff
@@ -1,13 +1,16 @@
 <ul role="tablist" class="umb-tabs-nav">
-    <li ng-click="vm.clickTab($event, tab)" class="umb-tab" role="tab" aria-selected="true" tabindex="0" ng-repeat="tab in vm.tabs |  limitTo: vm.maxTabs" data-element="tab-{{tab.alias}}" ng-class="{'umb-tab--active': tab.active, 'umb-tab--error': tabHasError}" val-tab>
-        <a>{{ tab.label }}</a>
+    <li class="umb-tab" ng-repeat="tab in vm.tabs |  limitTo: vm.maxTabs" data-element="tab-{{tab.alias}}" ng-class="{'umb-tab--active': tab.active, 'umb-tab--error': tabHasError}" val-tab>
+        <button class="btn-reset umb-tab-button" ng-click="vm.clickTab($event, tab)" role="tab" aria-selected="{tab.active}">{{ tab.label }}</button>
```

Comment:
Remember any buttons without a `type` attribute will just submit whatever form it might be in, need to always specify `type="button"`.

```suggestion
        <button type="button" class="btn-reset umb-tab-button" ng-click="vm.clickTab($event, tab)" role="tab" aria-selected="{tab.active}">{{ tab.label }}</button>
```

Reply:
Ah yeah good catch, i've updated the PR.